lib/create-test-action.js

```diff
@@ -18,20 +18,15 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node_version: ["12", "14", "16"]
+        node_version: ["14", "16", "18"]
```

Comment:
Has there been a statement about what node versions we're supporting? Are 14, 16, and 18 generally considered by the community to be the supported versions? (Sorry, I've been out of the Node ecosystem a little bit.)

Reply:
This is inline with all the other current JS octokit repositories.

v18 is the latest LTS release
v16 the prior LTS
v14 is the LTS before then, the latest LTS release to support ESM modules

From the history of the JS octokit modules, it seems it's always been the current LTS, and the 2 prior ones

We had a discussion about dropping NodeJS v10 in https://github.com/octokit/oauth-app.js/issues/299 and further in the discussion also decided on dropping v12